changelog.txt

```diff
@@ -3,9 +3,10 @@
  * Fixed a bug which might cause failure when loading the native library.
  * Fixed a bug which might trigger a timeout in Context.finalize().
  * Fixed a bug which might cause RealmObject.isValid() to throw an exception if the object is deleted.
- * Updated Realm core to version 0.89.7
+ * Updated Realm core to version 0.89.8
    - fixed a potential stack overflow issue which might cause a crash when encryption was used.
    - embedded crypto functions from OpenSSL into Realm dynamic lib to avoid random issues on some devices.
+   - throw RealmEncryptionNotSupportedException if the device doesn't support Realm encryption.
```

Comment:
Add details here, like: 
At least one device type (HTX One X) contain system bugs that prevents Realms encryption to function properly. This is now detected, and an exception is thrown when trying to open/create an encrypted Realm file. It's up to the application to catch this and decide if it's ok to proceed without encryption instead.